### ISSUE TITLE
Tab accessibility: cart

### ISSUE THREAD

**Issue by ghost** [DivanteLtd/next#393]
When I open the cart, when I tab I am navigating the page behind the cart modal, I should be able to navigate to the checkout from here using keyboard only

**Comment by Qrzy** [comment]
This is covered by latest a11y audit done by SFUI team: https://github.com/DivanteLtd/storefront-ui/issues/960
We're waiting for a new version fixing this accessibility issue.

**Comment by pspaczek** [comment]
@kulczycki, @Qrzy tested here https://storybook.storefrontui.io/?path=/story/pages-cart--common. You can do it now, but after go to checkout you jump outsied. We have fix for this issues https://github.com/DivanteLtd/storefront-ui/pull/1088. But it need my CR
